library/Recurly/V3/API/Account.hs

```diff
@@ -15,7 +16,7 @@ getAccountSubscriptions
   -> m (Client.Response (Either RecurlyApi.RecurlyError [Types.Subscription]))
 getAccountSubscriptions accountCode = Recurly.liftRecurly $ do
   request <- RecurlyApi.makeRequest
-    ["accounts", Types.accountCodeToRecurlyText accountCode, "subscriptions"]
+    ["accounts", into @Text $ into @RecurlyText.RecurlyText accountCode, "subscriptions"]
```

Comment:
Should this (and others) use `via` instead? 
```suggestion
    ["accounts", via @RecurlyText.RecurlyText accountCode, "subscriptions"]
```

Reply:
Does this work? I'll give it a try.

Reply:
It does work! I like it.